app/views/docs/production.phtml

```diff
@@ -13,38 +13,48 @@
 
 <h2><a href="/docs/production#limitConsoleAccess" id="limitConsoleAccess">Limit Console Access</a></h2>
 
-<p>By default, anyone can signup for your Appwrite server, create projects, and use your computing power. While this is great for testing around or running your Appwrite service in a network isolated environment, it is highly not recommended for public production use.</p>
+<p>Appwrite provides three different methods to limit access to your Appwrite console. You can either set a list of IPs or email address which users are allowed to signup from. You can choose one or multiple restriction methods to apply.</p>
```

Comment:
You mentioned 3 methods and then said "either .. or .. ". I didn't understand what is 3rd option. Might be my english.

Reply:
It doesn't to me either XD

Reply:
Fixed